src/includes/class-wordlift-post-to-jsonld-converter.php

```diff
@@ -124,7 +126,11 @@ public function convert( $post_id, &$references = array(), &$references_infos =
 		 *
 		 * @since 3.20.0
 		 */
-		if ( ! empty( $jsonld['@type'] ) && 'WebPage' !== $jsonld['@type'] ) {
+		if (
+			! empty( $jsonld['@type'] )
+			&& 'WebPage' !== $jsonld['@type']
+			&& ! in_array( $jsonld['@type'], $publisher_types, true )
```

Comment:
We're already in post-to-jsonld converter which handles only *article|web-page. Remove this check.